### PR TITLE
Remove unnecessary `IDE0060` suppressions

### PR DESCRIPTION
**Pull request by xtqqczze**
Since https://github.com/dotnet/roslyn/pull/66535, `IDE0060: Remove unused parameter` doesn't warn for throw-only methods.

### CODE REVIEW

**Review by stephentoub — APPROVED** [review]
LGTM once CI is green. Thanks.

**Comment by xtqqczze** [comment]
https://github.com/dotnet/runtime/commit/29829ba669d9f5f953729456aa33321000d1fb86: Build Analysis check has passed.

**Comment by xtqqczze** [comment]
@stephentoub Sorry, I force-pushed after your approval; I've reset back to 29829ba669d9f5f953729456aa33321000d1fb86.

**Review by stephentoub — APPROVED** [review]

**Pull request merged**
